src/main/java/net/twasi/obsremotejava/events/responses/TransitionListChangedResponse.java

```diff
@@ -0,0 +1,17 @@
+package net.twasi.obsremotejava.events.responses;
+
+import net.twasi.obsremotejava.requests.ResponseBase;
+
+public class TransitionListChangedResponse extends ResponseBase {
```

Comment:
I'm thinking about this more and I think it makes sense to complete this work now.  Let me get this done so we can include it.

Reply:
Nevermind, looks like this isn't included in the released version of `obs-websocket` yet.  We'll have to wait until that's in.  In the meantime I'll add the code that should work but leave it commented for now.

Current release: https://github.com/Palakis/obs-websocket/blob/4.8.0/src/WSEvents.cpp#L504

Upcoming changes: https://github.com/Palakis/obs-websocket/blob/4.x-current/src/WSEvents.cpp#L561